packages/components/src/components/prio-nav/prio-nav.component.ts

```diff
@@ -0,0 +1,266 @@
+   * @param items The items to cache the position for
+   */
+  private cacheItemPositions(items: SynNavItem[]) {
+    items.forEach(item => {
+      // We have to measure while the items are in the primary slot,
+      // else we will just get the placement in the priority menu
+      item.removeAttribute('slot');
+      const { right } = item.getBoundingClientRect();
+      item.dataset.right = right.toString();
+    });
+
+    this.itemPositionsCached = true;
+  }
+
+  /**
+   * Determines which items should be shown or hidden, depending on the current width
+   */
+  private handlePriorityMenu() {
+    const navItems = this.getSlottedNavItems();
+
+    if (!this.itemPositionsCached) {
+      this.cacheItemPositions(navItems);
+    }
+
+    // Get the widths of the horizontal nav and the priority menu
+    // We subtract the width of the priority menu to get the final width
+    const { width } = this.horizontalNav.getBoundingClientRect();
+    const { clientWidth } = this.priorityMenu;
+    const finalWidth = width - clientWidth;
+
+    // Cache the first item
+    let firstHiddenItemRightPos: number | undefined;
+
+    let visibleItems = 0;
+
+    // Save the position of all the elements in a cache
+    navItems.forEach(item => {
+      // Make sure to use the cache obtained in createGhostItems
+      const isHidden = firstHiddenItemRightPos || parseFloat(item.dataset.right!) >= finalWidth;
+
+      if (isHidden) {
+        item.removeAttribute('horizontal');
+        item.setAttribute('slot', 'menu');
+
+        // Makes sure the item is focusable in a syn-dropdown
+        item.setAttribute('role', 'menuitem');
+
+        // Get the position of the first item
+        // Will get used to position the priority menu
+        if (!firstHiddenItemRightPos) {
+          firstHiddenItemRightPos = parseFloat(item.dataset.right!);
+        }
+      } else {
+        visibleItems += 1;
+        item.setAttribute('horizontal', 'true');
+        item.removeAttribute('slot');
+        item.removeAttribute('tabindex');
+
+        // Reset the role to the original value
+        if (item.dataset.originalRole) {
+          item.setAttribute('role', item.dataset.originalRole);
+        } else {
+          item.removeAttribute('role');
+        }
+      }
+    });
+
+    // Tell the render call that we have items in the priority menu
+    // and toggle the visibility of the priority menu label
+    this.hasItemsInDropdown = visibleItems !== navItems.length;
+    this.amountOfVisibleItems = visibleItems;
+  }
+
+  private renderPriorityMenu() {
+    return html`
+      <syn-dropdown
+        class=${classMap({
+          'priority-menu': true,
+          'priority-menu--hidden': !this.hasItemsInDropdown,
+        })}
+        part="priority-menu"
+        placement="bottom-end"
+      >
+        <syn-nav-item slot="trigger" horizontal>
+          <syn-icon name="more" library="system" label="More" slot="prefix"></syn-icon>
```

Comment:
The `part`'s are missing here.
And don`t we also want to make the prio-menu-icon exchangeable via slot like in all other components? 
What do you think @mariohamann ?

Reply:
I talked about this with Mario. We came to the point, that this is a slot which is used for internal functionality and therefore should not be exposed with the official documentation

Reply:
i added `part`'s to all elements